orangecontrib/spectroscopy/widgets/owhyper.py

```diff
@@ -722,7 +831,7 @@ def update_view(self):
         self.data_imagepixels = None
         self.data_valid_positions = None
 
-        if self.data and self.attr_x and self.attr_y:
+        if self.data and self.attr_x and self.attr_y and self.parent.image_values():
```

Comment:
Why the additional check?

Reply:
`image_values` was changed to possibly return None to handle unique attrs problem, discussed below.